src/main/java/at/hannibal2/skyhanni/config/features/garden/NextJacobContestConfig.java

```diff
@@ -37,6 +38,16 @@ public class NextJacobContestConfig {
     @ConfigEditorBoolean
     public boolean fetchAutomatically = true;
 
+    @Expose
+    @ConfigOption(name = "Alternative Boosted Highlight", desc = "Highlight the current boosted crop with an highlight in addition to the enchant glint.")
```

Comment:
please use more clear names and description here. is this an alterantive? is this something different? is this additonally, or replaces the original/old highlight variant?